<launch>
  <node name="front_camera" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="camera_link" />
    <param name="autofocus" value="false" />
    <param name="io_method" value="mmap"/>
    <param name="camera_info_url" value="file://$(find eyecu)/config/logitech_webcam_calibration.yaml"/>
    <param name="camera_name" value="front_camera"/>
  </node>

</launch>
